docs/usage_guide/optimization_guide/index.rst

```diff
@@ -0,0 +1,58 @@
+Optimization Guide
+==================
+
+Here you can find information on Modin optimizations both for general query pipeline and for specific operations.
+
+Generic settings
+""""""""""""""""
+
+If you are working with a wide DataFrame and performing an operation that requires ``map`` Core Dataframe Algebra operator
+to be applied cell-wise, you may want to change the number of partitions DataFrame should partition into to avoid
+excessive amount of remote partitions, which is much greater than the number of CPUs.
+
+.. code-block:: python
+
+  from multiprocessing import cpu_count
+  import modin.config as cfg
+  import modin.pandas as pd
+  import numpy as np
+
+  data = np.random.randint(0, 100, size=(2**15, 2**15))
+
+  cfg.NPartitions.put(cpu_count())
```

Comment:
It might be useful to add an inline comment here with an actual `cpu_count()` number.

Reply:
added the `NPartitions` [variable printing](https://github.com/YarShev/modin/blame/dev/yigoshev-issue3863/docs/usage_guide/optimization_guide/index.rst#L81-L84) in the example